news/_posts/2020-05-13-the-new-order-pages-in-prestaShop-1.7.7.md

```diff
@@ -0,0 +1,225 @@
+
+- **Order's preview**: Opens a panel just below the line of the order to preview the order's essential information.
+
+![Order preview](/assets/images/2020/05/order_preview.gif "Order preview")
+
+***What is the goal***:
+
+Allows the merchant to check, prepare and track his order without having to open it. So the merchant won't have to open the order to check essential information that doesn't require actions.
+
+***How it works***: 
+
+When the user clicks on the icon, it displays the order's preview below the line selected. The order preview lists the following information: the invoice address, the shipping address, the customer details, the carrier details (and the tracking ID when available) and the products' list. 
+
+The products' list displays on each line the product name, reference, stock location (if available), quantity, tax price and total price for the products. The total price follows the customer group configuration. If the customer group sets the prices with or without tax, it impacts the displayed amount in the total column and adds the related label "tax excluded" or "tax included".
+
+The list displays the 10 firsts products. The remaining products can be seen by expanding the list.
+
+There is a new hook called displayOrderPreview on the bottom of the preview.
+
+![Order preview's hook](/assets/images/2020/05/hook-preview.png "Order preview's hook")
+
+It allows modules to add new elements on the preview. For more specific needs, the preview can be changed by overriding template, find (here)[https://devdocs.prestashop.com/1.7/modules/concepts/templating/admin-views/] more information.
+
+
+
+
+- **Editable order status labels:** Allows to change the order status directly from the listing.
+ 
+![Edit a status](/assets/images/2020/05/editable_order_statuses_labels.png "Edit a status")
+
+***What is the goal***:
+
+It allows the merchant fast management of the order fulfilment.
+
+***How it works***: 
+
+When the user clicks then on the status label, it displays a drop-down list with all of the available statuses. By selecting one of the items on the list, the user can update the order status.
+
+
+- **Statuses' colour**: Color according to the order status
+
+![Order statuses by category](/assets/images/2020/05/order_status_labelsx2.png "Order statuses by category")
+
+***What is the goal***: 
+
+The colours used by default to contextualize the status is based on a wide range of colours without following any real logic. To bring a structure and meaning to the statuses colours, the statuses are grouped in 4 colours categories:
+
+- orders waiting for a customer action as in waiting for a bank check in dark blue
+- orders waiting for a merchant action following an accepted payment in light blue
+- when no action is needed from the merchant as when the product is delivered in green
+- special status as payment error order in red or black
+
```

Comment:
```suggestion
- Light blue: orders waiting for a merchant action following an accepted payment 
- Green: no action needed from the merchant (i.e. when the product is delivered)
- Red or black: special status (i.e. payment error)

```

Reply:
We talk about 4 colors but actually use 5, yet only have 4 bullets points 🤔 
not clear
I put the colors at the beginning as they are the main topic